orchagent/aclorch.cpp

```diff
@@ -208,7 +224,23 @@ bool AclRule::validateAddMatch(string attr_name, string attr_value)
                     SWSS_LOG_ERROR("Failed to locate port %s", alias.c_str());
                     return false;
                 }
-                m_outPorts.push_back(port.m_port_id);
+                if (port.m_type == Port::PHY)
+                {
+                    m_outPorts.push_back(port.m_port_id);
+                }
+                else if (port.m_type == Port::LAG)
+                {
+                    m_outPorts.push_back(port.m_lag_id); 
```

Comment:
does not match SAI spec. 

https://github.com/opencomputeproject/SAI/blob/master/inc/saiacl.h#L1260

same as above.

Reply:
Thinks, we will delete LAG port.